list.yaml

```diff
@@ -200,3 +202,7 @@
     - online portal to view stored items
     - dependant on "network" i.e. shipping being cheap
 
+=======
```

Comment:
wtf is this shit @OstrichProjects you rebased incorrectly probably. just delete any changes to this file and ill approve